site/solutions/kustomize/README.md

```diff
@@ -0,0 +1,183 @@
+---
+title: "kpt and kustomize"
+linkTitle: "kustomize"
+type: docs
+---
+
+## Overview
```

Comment:
We should just include this in the README of the example package.

I prefer we don't add additional top-level sections to the kpt.dev until there are multiple solutions. Otherwise, this will stand out and people may overfit on it.

Reply:
I will be adding a ConfigSync section shortly.  That's another critical journey we need to cover.

Reply:
This seems to duplicating the content here:

https://github.com/GoogleContainerTools/kpt/tree/main/package-examples/kustomize-pkg

Per offline discussion, I suggest try to make the solution packages self-contained, and not requiring touching `/site` every time we add an example. For now, we can keep it simple, and just point to the package README from the FAQ. Long term, we can provide an index page, and render the package README.me in kpt.dev as well, similar to what we do for http://catalog.kpt.dev/

Reply:
I think we need to do a bit of homework about onramp guides for `kustomize` users. For ex. I would like to show a kustomize users how to accomplish variants using `fork/edit` patterns and highlight the fact some of the common customizations such as `labels`, `namespace`, `name changes` etc, how they are done using `pipeline` construct. And second aspect of onramp is what is being captured here where both tools co-exist, but that requires users to deal with two tools (not sure if that scales in a big organization).

So I agree with @frankfarzan here, before, putting a prominent solution page, we should form an informed opinion about the approaches. So for now having a brief FAQ entry pointing to the example pkg containing all the details in the README of the example package seems like a good idea. Also, we also need to vet the example with a few users before me make it prominent on the site.

Reply:
Sounds good.  I have vetted this pattern with a couple of customers.  I think if you'd prefer to take this into a different direction I am cool with it.  I would like to get back to one of the customers (2nd customer who asked for guidance) next week if possible.

Reply:
> I would like to get back to one of the customers (2nd customer who asked for guidance) next week if possible.

Can we move the content from `site/solutions/kustomize/README.md` to the `examples/kustomize-pkg/README.md` to unblock the customer communication? And from `FAQ` adding a link to that README ?